app/controllers/api/current/patients_controller.rb

```diff
@@ -46,17 +46,25 @@ def unscope_associations
   end
 
   def merge_if_valid(single_patient_params)
-    validator = Api::Current::PatientPayloadValidator.new(single_patient_params)
-    logger.debug "Patient had errors: #{validator.errors_hash}" if validator.invalid?
-    if validator.invalid?
-      NewRelic::Agent.increment_metric('Merge/Patient/schema_invalid')
-      { errors_hash: validator.errors_hash }
-    else
+    if FeatureToggle.enabled?('SKIP_API_VALIDATION')
```

Comment:
Let's maybe remove this code-change in favor of disabling it from this [place](https://github.com/simpledotorg/simple-server/blob/a4ea71b23b1881b0efabfdd3407ef3f30558d446/app/validators/api/current/patient_payload_validator.rb#L24) for a much cleaner disable?